packages/common/src/ArcGISContextManager.ts

```diff
@@ -432,7 +438,25 @@ export class ArcGISContextManager {
           ];
           break;
         case "serviceStatus":
-          this._serviceStatus = result as HubServiceStatus;
+          // eslint-disable-next-line no-case-declarations
+          const fetchedStatus = result as HubServiceStatus;
```

Comment:
this is kinda gross, but not using this cast made the subsequent lines inscruable for all the needed type-casting.